src/main/scanLogic/scanRunners/analyzerModels.ts

```diff
@@ -96,6 +97,8 @@ export interface FileRegion {
     startColumn: number;
     endColumn: number;
     snippet?: ResultContent;
+    tokenValidation?: string;
+    metadata?: string;
```

Comment:
there are no attribute like this in the SARIF spec. this should not be here (probably should move to the struct at `secretsScan.ts` or in `analyzerUtils.ts` - FileWithSecurityIssues)

Reply:
ok i changed to properties, FileWithSecurityIssues is not good for me (too much high level, I need per location)